funnel/templates/organization_membership.html.jinja2

```diff
@@ -1,4 +1,4 @@
-{% extends "index.html.jinja2" %}
+{% extends "profile.html.jinja2" %}
```

Comment:
I don't know if this is a good or bad idea.

IMO, profile layout should be a a separate template from the profile template, but it can also work to simply modify the existing template. I haven't encountered this before.